### ISSUE TITLE
"Related kata" link takes you to the /forks page

### ISSUE THREAD

**Issue by anter69** [Codewars/codewars.com#1620]
Clicking on the "related kata" link when reviewing translations takes you to the `/forks` page of the kata, which is pretty useless. It should take you to the `/discuss` page, as you usually use it to check issues/suggestions, and the kata description is already available via the "kata reference" button just above.

![image](https://user-images.githubusercontent.com/12795864/48137610-bf7cb300-e2a2-11e8-85e2-6f67e6b0acaa.png)

![image](https://user-images.githubusercontent.com/12795864/48137519-8f351480-e2a2-11e8-9d41-a43f7d3df0a8.png)

**Comment by kazk** [comment]
Yeah, I actually don't understand why it says "Related Kata" to begin with.
